modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -94,10 +96,14 @@ export function createOutputScript2of3(pubkeys: Buffer[], scriptType: ScriptType
  * Creates and returns a taproot output script using the user and bitgo keys for the aggregate
  * public key and a taptree containing a user+bitgo 2-of-2 script at the first depth level of the
  * tree and user+backup and bitgo+backup 2-of-2 scripts one level deeper.
- * @param pubkeys - a pubkey array containing the user key, backup key, and bitgo key in that order
+ * @param pubkeys - a pubkey triple containing the user key, backup key, and bitgo key in that order
  * @returns {{scriptPubKey}}
  */
-function createTaprootScript2of3([userKey, backupKey, bitGoKey]: [Buffer, Buffer, Buffer]): SpendableScript {
+function createTaprootScript2of3(pubkeys: Buffer[]): SpendableScript {
```

Comment:
This can still use the `Triple<Buffer>` type no? SInce it's only called internally and not exported and we're already calling `isTriple` in `createOutputScript2of3` with the proper type guards, we then wouldn't need to call it again below.